content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
+Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
+Wooh!
+
+Manubot applies **tactics from open source** software to manuscript writing.
+Manuscripts are written in markdown, using git for version control and GitHub for collaboration and review.
+Continuous integration automatically rebuilds and deploys the manuscript when it changes [@doi:10.1038/nbt.3780; @doi:10.1038/550143a].
+Manubot provides many state-of-the-art features.
+For example, users cite standard identifiers, while bibliographic details are automatically retrieved.
+
+Currently, the Manubot consists of two repositories: [`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) (a blank manuscript with the required files) and [`greenelab/manubot`](https://github.com/greenelab/manubot) (a backend Python package).
+As of December 2017, [10](https://github.com/greenelab/manubot-rootstock/graphs/contributors) individuals have contributed to the **Manubot codebase**, which has [33](https://github.com/greenelab/manubot-rootstock/stargazers) ⭐s on GitHub.
+For more information on the Manubot, see [these slides](http://slides.com/dhimmel/manubot#/) as well as the in-progress [Meta Review](https://greenelab.github.io/meta-review/).
+
+We initially created the Manubot to enable the **Deep Review**, a review paper on deep learning that we collaboratively drafted [on GitHub](https://github.com/greenelab/deep-review).
+Over 30 authors contributed to the Deep Review [@doi:10.1101/142760], using [issues](https://github.com/greenelab/deep-review/issues?utf8=%E2%9C%93&q=is%3Aissue) to discuss source material and [pull requests](https://github.com/greenelab/deep-review/pulls?utf8=%E2%9C%93&q=is%3Apr) to propose manuscript changes.
+The community appreciated the breadth of perspectives on a quickly-developing topic and the Deep Review was the most-viewed _bioRxiv_ preprint from April–June 2017 [@doi:10.1038/d41586-017-08493-x].
```

Comment:
Check the metrics for the others, but I think it was actually the most-viewed preprint from all of 2017.

Reply:
> most-viewed preprint from all of 2017

Interesting! Out of the four quarterly winners in the [Nature News article](https://doi.org/10.1038/d41586-017-08493-x), the Deep Review has the most abstract and PDF views as of December 21, 2017:

[Regulation of Life Span by the Gut Microbiota in The Short-Lived African Turquoise Killifish](https://www.biorxiv.org/content/early/2017/03/27/120980.article-metrics#details)
12,619 abstract views, 13,502 PDF views

[Opportunities And Obstacles For Deep Learning In Biology And Medicine](https://www.biorxiv.org/content/early/2017/05/28/142760.article-metrics#details)
29,235 abstract views, 22,573 PDF views

[Why Does the Neocortex Have Columns, A Theory of Learning the Structure of the World](https://www.biorxiv.org/content/early/2017/09/28/162263.article-metrics#details)
20,705 abstract views, 17,504 PDF views

[RNA velocity in single cells](https://www.biorxiv.org/content/early/2017/10/19/206052.article-metrics#details)
6,733 abstract views, 3,496 PDF views

I'm not sure whether the selection of the preprints in the Nature News article only counts views during that quarter, or instead counts all views up to query time. If the former, it's possible Deep Review is not the most viewed _bioRxiv_ preprint from 2017.

Reply:
My impression is that it is all views up to query time, although the wording is unclear. I have asked Richard Sever via twitter DM for a clarification.

Reply:
Ok, I have also sent him a link to this exact comment in case he prefers to reply via GitHub! 🤞